src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -160,7 +166,8 @@ class Browser extends DashboardView {
     let relation = this.state.relation;
     if (isRelationRoute && !relation) {
       const parentObjectQuery = new Parse.Query(className);
-      const parent = await parentObjectQuery.get(entityId, { useMasterKey: true });
+      const { useMasterKey } = this.state;
+      const parent = await parentObjectQuery.get(entityId, { useMasterKey: useMasterKey });
```

Comment:
You can use the object shorthand `{ useMasterKey }` everywhere below.

Reply:
Ok, used it on entire file here.